docs/index.rst

```diff
@@ -11,3 +11,6 @@ Contents:
    elasticsearch_connector
    configuration_options
    changelog
+
```

Comment:
same comments as https://github.com/confluentinc/schema-registry/pull/777#discussion_r181829904

Reply:
"This demo shows users how to deploy a Kafka streaming ETL" ... the "a" seems out of place. @joel-hamill, is that what you were referring to?

Reply:
@rhauch my comments were about the usage of "our" and "users".